.github/labeler.yml

```diff
@@ -0,0 +1,36 @@
+documentation:
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'docs/**'
+          - 'mkdocs.yml'
+docker:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/Dockerfile'
+          - 'docker-compose.yml'
+element:
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'apps/prairielearn/elements/**'
+python:
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'apps/prairielearn/python/**'
+          - 'apps/prairielearn/elements/**'
```

Comment:
I recommended specifically `apps/prairielearn/elements/**/*.py` for a reason: there are a bunch of non-Python files in those directories too (CSS, JSON, Mustache, JS).

Reply:
Ah, I see.